tt_metal/common/bfloat16.hpp

```diff
@@ -123,6 +123,19 @@ inline std::vector<std::uint32_t> create_arange_vector_of_bfloat16(uint32_t num_
     return vec;
 }
 
+inline std::vector<bfloat16> create_random_vector_of_bfloat16_nopack(uint32_t num_bytes, int rand_max_float, int seed, float offset = 0.0f) {
```

Comment:
I would instead rename create_random_vector_of_bfloat16 to create_random_vector_of_bfloat16_packed and call this function create_random_vector_of_bfloat16

Additionally, I believe the type of rand_max_float should be float, not int

Reply:
I'd postpone that, since the original function is used in some other examples, and I'm not the owner. I'll close the loop with @tt-rkim

Reply:
https://github.com/search?q=repo%3Atenstorrent-metal%2Ftt-metal%20create_random_vector_of_bfloat16&type=code